src/main/java/com/management/HumanResources/service/CreateService.java

```diff
@@ -48,4 +49,12 @@ public void initDefaultEmployeeTime(Employee emp) {
         firebase.addEmployeeTime(time);
     }
 
+    public boolean enterNewFeedback(Feedback feedback) {
+        if(feedback.getId() == 0){
```

Comment:
It is better to generate the feedback ID on the server side.

Reply:
The feedback ID is the same id as the employee, which the client would already have access to when this request is sent.

Reply:
Do we want to allow only one feedback submitting for an employee? How would you identify two feedback submissions from the same employee?

Reply:
That is a good point that you bring up. I might need to research a good way to tackle this. I made a follow up issue here for that #10